bundles/AdminBundle/Resources/public/js/pimcore/document/pages/settings.js

```diff
@@ -50,30 +54,37 @@ pimcore.document.pages.settings = Class.create(pimcore.document.settings_abstrac
                 this.metaDataPanel.updateLayout();
             }.bind(this);
 
-            this.metaDataPanel = new Ext.form.FieldSet({
-                title: t("html_tags") + " (&lt;meta .../&gt; &lt;link .../&gt; ...)",
-                collapsible: false,
-                autoHeight:true,
-                width: 700,
-                style: "margin-top: 20px;",
-                items: [{
-                    xtype: "toolbar",
-                    style: "margin-bottom: 10px;",
-                    items: ["->", {
-                        xtype: 'button',
-                        iconCls: "pimcore_icon_add",
-                        handler: addMetaData
+            var user = pimcore.globalmanager.get("user");
+            if (user.admin) {
```

Comment:
mhhh, not sure if it can be done in a minor release, eventually needs an upgrade note (for upgrading 10.5 to 10.6), something like: 'The HTML-Tags (document metadata) field under `SEO & Settings` panel is now only visible and editable by Admin users`.